assets/admin/editor-wizard/steps/lesson-details-step.js

```diff
@@ -21,6 +21,7 @@ import detailsStepImage from '../../../images/details-step.png';
  */
 const LessonDetailsStep = ( { data: wizardData, setData: setWizardData } ) => {
 	const { editPost } = useDispatch( editorStore );
+	const postTitle = usePostTitle();
```

Comment:
Feel free to skip if you don't like, but just a small suggestion:

Since we created a hook for that, we could move all the data part to the hook. It could be a hook receiving the `wizardData`, and returning `lessonTitle` and `updateLessonTitle`.

So the component would just have this hook call, and the `return`.

Reply:
Changed here: [77de975](https://github.com/Automattic/sensei/commit/77de97512acdcad4d0bfbc4c2bc0059179a66f87)